src/SFML/Graphics/Texture.cpp

```diff
@@ -441,6 +454,94 @@ void Texture::update(const Uint8* pixels, unsigned int width, unsigned int heigh
 
 
 ////////////////////////////////////////////////////////////
+void Texture::update(const Texture& texture)
+{
+    // Update the whole texture
+    update(texture, 0, 0);
+}
+
+
+////////////////////////////////////////////////////////////
+void Texture::update(const Texture& texture, unsigned int x, unsigned int y)
```

Comment:
I tried to stay consistent with the other update methods and they all take `unsigned int`s instead of `sf::Vector2u`s. I assume there was/is a reason for that, so the same reason will apply here as well.